content/docs/usage.md

```diff
@@ -1,70 +1,100 @@
 # Using CML
 
-You'll need a GitHub or GitLab account to begin. Users may wish to familiarize
-themselves with [Github Actions](https://help.github.com/en/actions) or
-[GitLab CI/CD](https://about.gitlab.com/stages-devops-lifecycle/continuous-integration/).
+A GitLab, GitHub, or Bitbucket account is required. Familiarity with
+[Github Actions](https://help.github.com/en/actions) or
+[GitLab CI/CD](https://about.gitlab.com/stages-devops-lifecycle/continuous-integration)
+may also be beneficial.
 Here, will discuss the GitHub use case.
```

Comment:
This remark may be a bit outdated, given that we provide separate headings for each platform.